pkg/config/provider.go

```diff
@@ -44,7 +44,9 @@ func (cc ResourceConfiguratorChain) Configure(r *Resource) {
 // and controllers. Typically, we expect to see ProviderConfig packages here.
 type BasePackages struct {
```

Comment:
Is this `BasePackages` the same as the config package we configured with `PackageNameConfig`? 
Could we name that variable as `PackageNameBase="config"` or at least have a comment indicating this?

Reply:
Done. Renamed the `const` from `PackageNameConfig` to `PackageNameFamily`.